docs/certificate-key/settings/oid.md

```diff
@@ -0,0 +1,25 @@
+By definition, object identifiers or `OID`s are an identifier mechanism standardised by the International Telecommunication Union (ITU) and ISO/IEC for naming any object, concept, or "thing" with a globally unambiguous persistent name. `OID`s are in format of tree hierarchy, where each node is represented by number and separated by dot. 
```

Comment:
I do not think this is true, OIDs can be standardized by different means, and can be defined also for private use.